tests/test_acceptance/test_graph/test_tesserae.py

```diff
@@ -44,3 +46,6 @@ def test_mosaic_alignment_on_medium_queries_and_two_templates(self):
             assert p[2][1] == repeat(" ", partial_len) + targets[1][partial_len:total_len]
             assert p[2][2] == partial_len
             assert p[2][3] == total_len - 1
+
+            assert p == q
+            assert t1.llk == t2.llk
```

Comment:
I think that testing that the `mem_limit=True` option does not change results should be a separate test. This test is already quite long, and it does not look like the new assert statements in any way depend on the nine assert statements preceding them.

Reply:
You are completely right. Will do this.